apps/webservice/src/app/[workspaceSlug]/(app)/_components/deployment-resource-drawer/TableRow.tsx

```
@@ -0,0 +1,352 @@
+              return (
+                <ReleaseJobTriggerChildRow
+                  release={release}
+                  releaseJobTrigger={trigger}
+                  environment={environment}
+                  deployment={deployment}
+                  resource={resource}
+                  key={trigger.id}
+                />
+              );
+            })}
+          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)